src/components/social-count/SocialCountSection.jsx

```diff
@@ -0,0 +1,26 @@
+import React, {useEffect, useState} from 'react';
+import {getWishlistSocialCount} from '../../swym/store-apis';
+import './SocialCountSection.css';
+
+export default function SocialCountSection(productData) {
+  const [count, setCount] = useState(0);
+  
+  useEffect(() => {
+    async function setSocialCount(){
+        try {
+            const res = await getWishlistSocialCount(productData.productData.product.id.split('/')[4]);
+            setCount(res.data.count);
```

Comment:
@gouravswym we should check `res.status` first and also check res.data.count is not undefined and is number. Else UI will render undefined.